patches/wpewebkit/0002-Define-MESA_EGL_NO_X11_HEADERS-when-not-using-GLX.patch

```diff
@@ -0,0 +1,13 @@
+--- a/Tools/wpe/backends/ViewBackend.cpp	2018-10-04 10:04:52.264448058 +0000
++++ b/Tools/wpe/backends/ViewBackend.cpp	2018-10-04 10:06:02.232680294 +0000
+@@ -25,6 +25,10 @@
+ 
+ #include "ViewBackend.h"
+ 
++#if !defined(USE_GLX) || !USE_GLX
++// This define is used in mesa and other GL implementations to exlcude X11 headers.
++#define MESA_EGL_NO_X11_HEADERS 1
```

Comment:
I think i added this by mistake because I am not using MESA, I am using RPI_USERLAND. But in between I had tried MESA and may be that time I added this Patch. I will clean it up.

Reply:
@aperezdc  Have you tried enabling MiniBrowser in current buildroot overlay ? While trying to build MiniBrowser with FDO backend I got X11 header error.

Like you suggested adding #include <wayland-egl.h> before #include <epoxy/egl.h> will fix the error. Same thing is done in file Tools/wpe/backends/WindowViewBackend.cpp but not in Tools/wpe/backends/ViewBackend.cpp .. Shall I update this patch to include waylang-egl.h